src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -10,208 +10,252 @@ import * as url from 'node:url';
-
-            let bytesReceived = 0;
 
-            // Set up progress tracking
-            if (onProgress && typeof response.on === 'function') {
-                if (!isNaN(totalBytes)) {
-                    console.log(`Total download size: ${(totalBytes / 1024 / 1024).toFixed(2)} MB`);
-                }
+                    // Setup progress tracking manually since onDownloadProgress doesn't work with streams
+                    let bytesReceived = 0;
+                    let lastReportedPercent: number | null = null;
+                    let progressThrottleMs = options.progressThrottleMs || 200;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'progressThrottleMs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)